src/webgpu/shader/validation/wgsl/runtime-array-not-last-v2.fail.wgsl

```diff
@@ -0,0 +1,12 @@
+# v-0015 - This fails because of the runtime array is not last member of the struct.
```

Comment:
Maybe mention the array is aliased to show the difference to runtime-array-not-last.fail.wgsl